cmd/config.go

```diff
@@ -309,6 +301,82 @@ type CAADistributedResolverConfig struct {
 	Proxies     []string
 }
 
+// TemporalLogDescription describes a single shard of a temporally shared
```

Comment:
*sharded. Also I'm getting nitpicky here, but can we rename this `LogShard`? I think the fact that `TemporalSet` and `TemporalLogDescription` start with the same big long word, it's hard to distinguish them in my mind.